### ISSUE TITLE
After 166000 trianing steps the French to English translation is not improving

### ISSUE THREAD

**Issue by frajos100** [tensorflow/nmt#328]
After 166000 trianing steps the French to English translation is not improving. Is there any setting that needs to be done for better translation output?

French :
Marie détestait habituellement lundi matin, mais cette année était différente. Thomas était dans sa classe d'art et elle aimait Thomas. Elle a attendu à l'extérieur de la classe quand son amie Camille est arrivée.
Translation should be:
Marie usually hated Monday morning, but this year was different. Thomas was in her art class and she liked Thomas. She waited outside the classroom when her friend Camille arrived.
NMT translation:
@96k steps: Marie <unk> usually <unk> on Monday , but this year was <unk> Thomas was in her middle-class class and she <unk> <unk> she waited for the class when his <unk> <unk> was released .
@166k steps: Marie <unk> <unk> <unk> , but this year was <unk> Thomas was in his favorite class and she <unk> <unk> she waited at <unk> of the class when her <unk> <unk> was <unk> .

**Comment by hackerwei** [comment]
@frajos100 
how to prepare the French -English train set? I do not find any script to preprocess French-English.

**Comment by frajos100** [comment]
You would need to modify the German-English script to download the files for the Franc to English repository.

**Comment by hpulfc** [comment]
make sure that the corpus is cleaning.
also,  you can consider that use coverage. Unfortunately , it hard to implement it !!!

**Comment by luozhouyang** [comment]
You can try to use a different learning rate decay strategy.
